src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -178,45 +178,52 @@ private void GetFiles(JObject fileObject, List<string> files)
 
             (string name, string version) = LibraryNamingScheme.Instance.GetLibraryNameAndVersion(libraryNameStart);
 
+            int at = name.IndexOf('@');
```

Comment:
Can you add a TODO here to track that the LibraryNamingScheme should not be returning "{name}@" as the name?